### PR TITLE
[candi] replace logrus core modules

### PR DESCRIPTION
**Pull request by sprait**
## Description

Replace Logrus to Slog implementation in next components:

- control-plane-manager
- dhctl (also delete simple logger, until our slog implementation will be support not only json formatter)  
- openvpn (migrator)
- registry-packages-proxy
- node-local-dns

## Why do we need it, and what problem does it solve?

Logging standardization.

## What is the expected result?

<details>
  <summary>examples logs</summary>

```
[deckhouse] root@facb1f0ca5a2 / # dhctl bootstrap --logger-type=simple  --ssh-user=ubuntu --ssh-agent-private-keys=/tmp/.ssh/id_rsa_vd --config=/config.yml
enum value must be one of pretty,json, got 'simple'

[deckhouse] root@facb1f0ca5a2 / # dhctl bootstrap --logger-type=json  --ssh-user=ubuntu --ssh-agent-private-keys=/tmp/.ssh/id_rsa_vd --config=/confi2.yml
{"level":"info","msg":"Debug log file: /tmp/dhctl/bootstrap-20241209072943.log\n","time":"2024-12-09T07:29:43Z"}
{"level":"info","msg":"State directory: /tmp/dhctl/fdce9d2e3852df847fc9cdf0dc943d1d9a99fb46b356f1fd071d6a7f1367d123\n","time":"2024-12-09T07:29:43Z"}
{"level":"info","msg":"[========================================================================================\n _____             _     _                                ______                _ _____\n(____ \\           | |   | |                              / _____)              | (_____)\n _   \\ \\ ____ ____| |  _| | _   ___  _   _  ___  ____   | /      ____ ____   _ | |  _\n| |   | / _  ) ___) | / ) || \\ / _ \\| | | |/___)/ _  )  | |     / _  |  _ \\ / || | | |\n| |__/ ( (/ ( (___| |\u003c (| | | | |_| | |_| |___ ( (/ /   | \\____( ( | | | | ( (_| |_| |_\n|_____/ \\____)____)_| \\_)_| |_|\\___/ \\____(___/ \\____)   \\______)_||_|_| |_|\\____(_____)\n========================================================================================]","time":"2024-12-09T07:29:43Z"}
{"level":"info","msg":"Cluster UUID","action":"start","process":"bootstrap","time":"2024-12-09T07:29:43Z"}
{"level":"info","msg":"Generated cluster UUID: c25f2286-0b9c-46bd-a26b-e3f57005e24f\n","time":"2024-12-09T07:29:43Z"}
{"level":"info","msg":"Cluster UUID","action":"end","process":"bootstrap","time":"2024-12-09T07:29:43Z"}
{"level":"info","msg":"Global preflight checks","action":"start","process":"common","time":"2024-12-09T07:29:43Z"}
{"level":"info","msg":"Checking PublicDomainTemplate is correctly","action":"start","process":"default","time":"2024-12-09T07:29:43Z"}
{"level":"info","msg":"Succeeded!\n","status":"SUCCESS","time":"2024-12-09T07:29:43Z"}
{"level":"info","msg":"Checking PublicDomainTemplate is correctly","action":"end","process":"default","time":"2024-12-09T07:29:43Z"}
{"level":"info","msg":"Checking registry credentials are correct","action":"start","process":"default","time":"2024-12-09T07:29:43Z"}
{"level":"info","msg":"Succeeded!\n","status":"SUCCESS","time":"2024-12-09T07:29:43Z"}
{"level":"info","msg":"Checking registry credentials are correct","action":"end","process":"default","time":"2024-12-09T07:29:43Z"}
{"level":"info","msg":"Global preflight checks","action":"end","process":"common","time":"2024-12-09T07:29:43Z"}
...
```

```
[root@dev-master-0 ~]# kubectl -n kube-system  logs -f stale-dns-connections-cleaner-77f4w
{"level":"info","msg":"Start","time":"2024-12-03T09:46:51Z"}
{"level":"info","msg":"This is a workaround for issue https://github.com/cilium/cilium/issues/31012.","time":"2024-12-03T09:46:51Z"}
{"level":"info","msg":"When both the cni-cilium and node-local-dns modules are enabled, and the node-local-dns pod has been restarted, stale DNS connections may occur.","time":"2024-12-03T09:46:51Z"}
{"level":"info","msg":"This is due to the UDP socket remaining active in the application pods with the destination IP address of the old node-local-dns pod, which has already been deleted.","time":"2024-12-03T09:46:51Z"}
{"level":"info","msg":"To prevent this problem, the following actions are taken:","time":"2024-12-03T09:46:51Z"}
{"level":"info","msg":"- Obtain the name and PodCidr of the node where the application is running.","time":"2024-12-03T09:46:51Z"}
{"level":"info","msg":"- Then every 30 seconds:","time":"2024-12-03T09:46:51Z"}
{"level":"info","msg":"  - Retrieve the current IP address of the node-local-dns pod.","time":"2024-12-03T09:46:51Z"}
{"level":"info","msg":"  - Retrieve all UDP sockets on the node and search for those with dst_port 53 and dsp_ip belonging to PodCidr, but not equal to the current IP address of the node-local-dns pod.","time":"2024-12-03T09:46:51Z"}
{"level":"info","msg":"  - If such sockets are found, delete them.","time":"2024-12-03T09:46:51Z"}
{"level":"info","msg":"kubeClient successfully inited","time":"2024-12-03T09:46:51Z"}
{"level":"info","msg":"The current node name is dev-caps","time":"2024-12-03T09:46:51Z"}
{"level":"info","msg":"Address: 127.0.0.1:8001","time":"2024-12-03T09:46:51Z"}
{"level":"info","msg":"Checks interval: 30s","time":"2024-12-03T09:46:51Z"}
{"level":"info","msg":"podCIDR on node dev-caps is 10.111.3.0/24","time":"2024-12-03T09:46:51Z"}
```

```
root@n-gorbatov-dev-fl-master-0:~# kubectl  -n kube-system  logs d8-control-plane-manager-zs7rx
Defaulted container "control-plane-manager" out of: control-plane-manager, image-holder-etcd, image-holder-kube-apiserver129, image-holder-kube-apiserver-healthcheck, image-holder-kube-controller-manager129, image-holder-kube-scheduler129
{"level":"info","msg":"check desired kubernetes version 1.29 against allowed kubernetes version list: 1.27,1.28,1.29,1.30,1.31","time":"2024-12-09T07:48:01Z"}
{"level":"info","msg":"phase: remove orphan files from dir /etc/kubernetes/deckhouse/kubeadm/patches","time":"2024-12-09T07:48:02Z"}
{"level":"info","msg":"phase: annotate node n-gorbatov-dev-fl-master-0 with annotation control-plane-manager.deckhouse.io/waiting-for-approval","time":"2024-12-09T07:48:02Z"}
{"level":"info","msg":"phase: waiting node node n-gorbatov-dev-fl-master-0 approval with annotation control-plane-manager.deckhouse.io/approved","time":"2024-12-09T07:48:02Z"}
{"level":"info","msg":"waiting for control-plane-manager.deckhouse.io/approved annotation on our node n-gorbatov-dev-fl-master-0","time":"2024-12-09T07:48:02Z"}
{"level":"info","msg":"waiting for control-plane-manager.deckhouse.io/approved annotation on our node n-gorbatov-dev-fl-master-0","time":"2024-12-09T07:48:03Z"}
{"level":"info","msg":"phase: waiting for all image-holder containers will be ready","time":"2024-12-09T07:48:03Z"}
{"level":"info","msg":"phase: waiting for all image-holder containers will be ready","time":"2024-12-09T07:48:13Z"}
{"level":"info","msg":"phase: check etcd manifest /etc/kubernetes/manifests/etcd.yaml","time":"2024-12-09T07:48:13Z"}
{"level":"info","msg":"phase: check kubelet config /etc/kubernetes/kubelet.conf","time":"2024-12-09T07:48:13Z"}
{"level":"info","msg":"phase: install kubeadm configuration","time":"2024-12-09T07:48:13Z"}
{"level":"info","msg":"file /etc/kubernetes/deckhouse/kubeadm/config.yaml is not changed, skipping","time":"2024-12-09T07:48:13Z"}
{"level":"info","msg":"file /etc/kubernetes/deckhouse/kubeadm/patches/etcd.yaml is not changed, skipping","time":"2024-12-09T07:48:13Z"}
{"level":"info","msg":"backup /etc/kubernetes/deckhouse/kubeadm/patches/kube-apiserver.yaml file","time":"2024-12-09T07:48:13Z"}
{"level":"info","msg":"install file /config/..2024_12_09_07_48_01.1961942233/kube-apiserver.yaml.tpl to destination /etc/kubernetes/deckhouse/kubeadm/patches/kube-apiserver.yaml","time":"2024-12-09T07:48:13Z"}
{"level":"info","msg":"file /etc/kubernetes/deckhouse/kubeadm/patches/kube-controller-manager.yaml is not changed, skipping","time":"2024-12-09T07:48:13Z"}
{"level":"info","msg":"file /etc/kubernetes/deckhouse/kubeadm/patches/kube-scheduler.yaml is not changed, skipping","time":"2024-12-09T07:48:13Z"}
```

</details>

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: dhctl
type: chore
summary: Replace Logrus to slog implementation, delete 'simple' logger.
impact_level: default
```
```changes
section: control-plane-manager, openvpn, registry-packages-proxy, node-local-dns
type: chore
summary: Replace Logrus to slog implementation.
impact: Restart components.
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/yandex-cloud'...

File: dhctl/go.mod [hunk 1]
```diff
@@ -118,6 +117,7 @@ require (
 	github.com/prometheus/procfs v0.15.1 // indirect
 	github.com/russross/blackfriday/v2 v2.1.0 // indirect
 	github.com/shopspring/decimal v1.3.1 // indirect
+	github.com/sirupsen/logrus v1.9.3 // indirect
```
Comment (name212) [inline dhctl/go.mod:120]: logrus was not deleted fully

**Review by name212 — CHANGES_REQUESTED** [review]
logrus was not deleted fully

File: modules/039-registry-packages-proxy/images/registry-packages-proxy/src/go.mod [hunk 2]
```diff
@@ -49,6 +50,7 @@ require (
 	github.com/prometheus/client_model v0.5.0 // indirect
 	github.com/prometheus/common v0.48.0 // indirect
 	github.com/prometheus/procfs v0.12.0 // indirect
+	github.com/sirupsen/logrus v1.9.3 // indirect
```
[the remark this answers is not in the document]
Comment (sprait) [reply modules/039-registry-packages-proxy/images/registry-packages-proxy/src/go.mod:53]: we discussed and decided that we would not avoid  inderect dependencies.

**Review by name212 — APPROVED** [review]

**Pull request merged**